flink-connector/flink/src/main/java/org/apache/gravitino/flink/connector/paimon/PaimonPropertiesConverter.java

```diff
@@ -37,44 +34,38 @@ private PaimonPropertiesConverter() {}
 
   @Override
   public Map<String, String> toGravitinoCatalogProperties(Configuration flinkConf) {
```

Comment:
is `toGravitinoCatalogProperties` and `toFlinkCatalogProperties` still usable in PaimonPropertiesConverter?

Reply:
fixed